### ISSUE TITLE
bower:install task throws cannot find module 'grunt' warning.

### ISSUE THREAD

**Issue by erictaylor** [linemanjs/lineman-bower#9]
As of version 0.0.4 running `lineman run` issues the following warning:

```
Running "bower:install" (bower) task
Warning: Cannot find module 'grunt' Used --force, continuing.
```

I am guessing this is related to the changes in commit [3a65e3e](https://github.com/linemanjs/lineman-bower/commit/3a65e3e670f833ee84cffa5cf713d24ed6de0e5b) on line 23.

**Comment by jayharris** [comment]
What versions of `lineman` and `node` are you running?

**Comment by erictaylor** [comment]
Lineman: v0.33.4
Node: v0.10.26

**Comment by jayharris** [comment]
@Psiablo I cannot reproduce this.

Running the following tasks against `lineman@0.33.4` works successfully. The build completes without errors.

```
lineman new sampleProject
cd sampleProject
npm install lineman-bower --save-dev
printf "{ \"name\": \"sampleProject\" }" > bower.json
bower install jquery --save
lineman build
```

The fourth line, `printf "{ \"name\": \"sampleProject\" }" > bower.json`, is just a shortcut to simulate `bower init`. I am substituting `init` just because it is interactive, and I want to proceed with a repeatable, non-interactive script.

Would you happen to have a sample repository that you can provide to demonstrate the error?
Could you run the script above and let me know what you get?

**Comment by jayharris** [comment]
@Psiablo Nevermind. I found the issue. I does not manifest for me because I have `grunt` installed globally. Once I uninstall it, I can reproduce the issue.

A fix is coming shortly.

**Comment by jayharris** [comment]
Fixed in v0.0.5. Published to `npm`
